tests/stress_tests/scripts/get_testdata.py

```diff
@@ -209,7 +209,14 @@ def main():
     # Do it manually to have only one folder with IRs
     for ir_src_path in args.omz_models_out_dir.rglob("*.xml"):
         ir_dst_path = args.omz_irs_out_dir / os.path.relpath(ir_src_path, args.omz_models_out_dir)
-        shutil.copytree(ir_src_path.parent, ir_dst_path.parent)
+
+        for item in os.listdir(ir_src_path.parent):
```

Comment:
@just-sparta what is the root cause of a problem? This solution is look like a workaround, but not a common solution